tracing-attributes/src/lib.rs

```diff
@@ -132,11 +132,14 @@ mod expand;
 /// Setting the level for the generated span:
 /// ```
 /// # use tracing_attributes::instrument;
-/// #[instrument(level = "debug")]
+/// #[instrument(level = Level::DEBUG)]
 /// pub fn my_function() {
 ///     // ...
 /// }
 /// ```
+/// Levels can be specifide either with `Level` constants, literal strings
```

Comment:
typo:
```suggestion
/// Levels can be specified either with `Level` constants, literal strings
```

Reply:
also, this is a tiny nit, but it might be worth making `Level` a link to the type's docs here?

Reply:
Hm, since `tracing-attribute` is a proc macro without a direct dependency on `tracing` the doc reference doesn't resolve. Is there a way to have a doc-only dependency?

Reply:
Sorry, should've replied. We probably shouldn't depend on `tracing-core`, but I'll change this PR to link to link to a docs.rs link for Level.